layouts/partials/navigation.html

```diff
@@ -5,26 +5,18 @@
       {{ $logoAlt:= site.Params.logo_alt }}
       {{ $orgName:= site.Params.brand.parent_org_name }}
       {{ $orgUrl:= site.Params.brand.parent_org_url }}
-      <a class="navbar-brand px-2"
-        href="{{ if (or $logo $logoAlt) }}{{ $orgUrl }}{{else}}{{ site.BaseURL | relLangURL }}{{ end }}">
+
+      <div class="text-center">
+        <a class="navbar-brand" href="{{ if (or $logo $logoAlt) }}{{ $orgUrl }}{{ else }}{{ site.BaseURL | relLangURL }}{{ end }}">
```

Comment:
It looks like this fixes the issue with the navigation bar, with or without a logo. 👍🏻 

![image](https://user-images.githubusercontent.com/4721034/181860103-93df91b7-9562-48ca-b45c-b3c67dd8eae0.png)

![Screenshot 2022-07-30 at 01-27-00 UNICEF O S  Inventory](https://user-images.githubusercontent.com/4721034/181860208-8fff1206-6721-495e-8750-0d025eda7abf.png)

Reply:
Resolved in commit 1f42c23b9bdf850505370e28efae7d3d15c899b1.